src/components/FilterSex.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import { FilterType } from '../types/enum';
+
+interface Props {
+  sexFilter: string,
+  sexFilterHandler: (value: FilterType) => void,
+}
+
+export const FilterSex: React.FC<Props> = ({
+  sexFilter,
+  sexFilterHandler,
+}) => {
+  const sexes = ['All', 'Male', 'Female'];
```

Comment:
Avoid using a non-descriptive variable name like 'sexes'. Consider using 'sexOptions' instead. And it's better to move this constant outside the component. And use the enum for string values. Seems that you've already had the enum for it
```
const sexOption = ...

const FilterSex = ...